src/shared/components/ncTable/NcTable.vue

```diff
@@ -225,15 +231,17 @@ export default {
 		},
 	},
 	mounted() {
-		subscribe('tables:selected-rows:deselect', this.deselectRows)
+		subscribe('tables:selected-rows:deselect', tableId => this.deselectRows(tableId))
```

Comment:
I think we also need to take `isView` into account here to not run into a conflict when there are views and tables with the same id

Reply:
Good catch, thanks